tasks/28_vowel-count/vowel-count.js

```diff
@@ -0,0 +1,10 @@
+export function getVowelCount(str) {
+  return [].reduce.call(str, (count, curr) => {
+    if (curr === 'a' || curr === 'e' ||
+           curr === 'i' || curr === 'o' ||
+           curr === 'u') {
+      return ++count;
+    }
+    return count;
+  }, 0);
```

Comment:
Just wondering if this would be readable?
```
return [...str].filter((el) => ['a', 'e', 'i', 'o', 'u'].includes(el)).length;
```

Reply:
Changed.